relay-config/src/config.rs

```diff
@@ -864,6 +864,11 @@ fn spool_envelopes_max_envelope_delay_secs() -> u64 {
     24 * 60 * 60
 }
 
+/// Default refresh frequency in ms for the disk usage monitoring.
+fn spool_disk_usage_refresh_frequency_ms() -> u64 {
+    100
```

Comment:
Should we default this to 1x per second? To get in the same ball park as the default memory check interval.

Reply:
It's set to 100ms for memory.